config/environment_indicator.indicator.yml

```diff
@@ -0,0 +1,3 @@
+name: Prod
+fg_color: '#ffffff'
+bg_color: '#6937ac'
```

Comment:
Should this be set explicitly? When I load this, I see purple "Prod" on my local

Reply:
I had set it in the DB last week, and intentionally not updated the config to avoid showing it on local envs.